From be98e47d4f08f054223bea2bff3d2db6c5257c26 Mon Sep 17 00:00:00 2001
From: Jonathan DE CESCO <jonathan.de.cesco@intel.com>
Date: Fri, 14 Sep 2012 11:16:09 +0200
Subject: [PORT FROM MAIN] wifi: [TI R5.00.21] fix TI R5.00.21 version

BZ: 57335

Align version.h to 0.21

Change-Id: I2ba470b1b33ad88a8f59ec4e18cbabfb3a5681de
Orig-Change-Id: Ib36bf52764a6c52a8631c5ef33af884d02ad9001
Signed-off-by: Jonathan DE CESCO <jonathan.de.cesco@intel.com>
Reviewed-on: http://android.intel.com:8080/66509
Reviewed-by: Trivelly, Jean <jean.trivelly@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
Reviewed-on: http://android.intel.com:8080/77982
Reviewed-by: Li, XiaojingX <xiaojingx.li@intel.com>
Tested-by: Li, XiaojingX <xiaojingx.li@intel.com>
---
 .../drivers/net/wireless/wl12xx/version.h          |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/wl12xx-compat/drivers/net/wireless/wl12xx/version.h b/wl12xx-compat/drivers/net/wireless/wl12xx/version.h
index 114768d..019f994 100644
--- a/wl12xx-compat/drivers/net/wireless/wl12xx/version.h
+++ b/wl12xx-compat/drivers/net/wireless/wl12xx/version.h
@@ -1,2 +1,2 @@
 static const char *wl12xx_timestamp = __TIMESTAMP__;
-static const char *wl12xx_git_head = "TI_R5.00.18";
+static const char *wl12xx_git_head = "TI_R5.00.21";
-- 
1.7.4.1

